src/apps/logging/logging.cpp

```diff
@@ -200,6 +200,7 @@ namespace loggingapp
       };
       make_endpoint(
         "log/private/prefix_cert", HTTP_POST, log_record_prefix_cert)
+        .set_auto_schema<LoggingRecord::In, bool>()
```

Comment:
These endpoints are not listed in `/api` if they don't have a schema.

Reply:
That sounds like a bug, we can and should at least have an entry with the path, the verb etc

Reply:
https://github.com/microsoft/CCF/pull/1738